dist-persist/wbstack/src/Internal/ApiWbStackInitMainPage.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace WBStack\Internal;
+
+/**
+ * This class contains the text for a default wiki Main Page.
+ */
+
+class ApiWbStackInitMainPage {
+    const TEXT = <<<EOF
```

Comment:
suggestion for the future: if this changes frequently, we should look into reading this content from a file and managing its contents in `wbaas-deploy`. For now, I think it's fine to keep it here.

Reply:
not entirely sure how that would work but yes I definitely agree, if this changes frequently another place would be better

Reply:
You can mount a ConfigMap as a file: https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/

i.e. we'd have

```yaml
message: |
  foo bar baz
  == whatever ==
```
 in the values file and the chart would make sure it ends up in `/run/message.txt` or similar.

Anyways, nothing to work on right now.